python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -1795,3 +1795,65 @@ def _get_optimize_pass(self):
             else:
                 pass
         return opt_ops, params_grads
+
+    @staticmethod
+    def append_bn_repeat_init_op(main_prog, startup_prog, num_repeats):
```

Comment:
Is there any way to avoid this API? This is a static method. Can you put it in contrib?

Reply:
Not currently until we find a way to modify startup program in passes.